index.d.ts

```diff
@@ -1,8 +1,21 @@
 import {PackageJson} from 'type-fest';
-import {Options as MinimistOptions} from 'minimist-options';
 
 declare namespace meow {
-	interface Options<Flags extends MinimistOptions> {
+	type FlagType = 'string' | 'boolean' | 'number';
+
+	interface Flag<Type extends FlagType, Default> {
+		readonly type?: Type;
+		readonly alias?: string;
```

Comment:
Considering #132, we may change like this:

```suggestion
		readonly alias?: string | ReadonlyArray<string>;
```

Reply:
That should be done in that PR though.

Reply:
Agreed. 👍